src/core/public/chrome/ui/header/nav_link.test.tsx

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
The license could be simplified to 
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```
since this is a newly added file.